File: CCC-10-J1.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int n;
    scanf("%d", &n);
    int count = 0;
    for(int i = n; i > 0; i--) {
        if(i <= 5 && ((i > n/2 && n % 2 == 1) || (i >= n/2 && n % 2 == 0))) {
            count++;
        }
    }
    printf("%d\n", count);

    return 0;
}

File: CCC-10-J1.py
n = input()
count = 0
for i in range(n, 0, -1):
    if(i <= 5 and ((i > n/2 and n % 2 == 1) or (i >= n/2 and n % 2 == 0))):
        count += 1
print count
